xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamBluray.cpp

```diff
@@ -1130,7 +1135,11 @@ bool CDVDInputStreamBluray::IsInMenu()
 {
   if(m_bd == nullptr || !m_navmode)
     return false;
-  if(m_menu /*|| !m_planes[BD_OVERLAY_IG].o.empty()*/)
+  
+  // since there is no way to tell in a BD-J blu-ray when a popup menu actually is visible,
+  // we have to assume that the blu-ray is in menu/navigation mode when there is an overlay
+  // on screen, even if it might be invisible (which is impossible to detect)
+  if(m_menu || m_hasOverlay /*|| !m_planes[BD_OVERLAY_IG].o.empty()*/)
```

Comment:
remove commented block

Reply:
I wonder if this code block would actually make `m_hasOverlay` obsolete, but I didn't dare to test it and just went with what fandangos used to figure out if there is an overlay.

Reply:
I don't mean the comment block but the ` /*|| !m_planes[BD_OVERLAY_IG].o.empty()*/` :)

Reply:
I know